Node/package.json

```diff
@@ -23,6 +23,7 @@
   "main": "./lib/botbuilder-azure.js",
   "typings": "./lib/botbuilder-azure.d.ts",
   "dependencies": {
+    "@types/tedious": "^1.8.31",
```

Comment:
This should be in devDependencies? :)

Reply:
Actually we were both wrong, as sheerun noted, so I reset it to maintain this depdency.